scripts/generate_pinval/generate_pinval.py

```diff
@@ -3,17 +3,18 @@
 Generate PINVAL report markdown (and optionally HTML) files for a given model
 run‑id. A user may ask for either **one or more explicit PINs** *or* for **all
 PINs that belong to a triad** (city, north, south). Exactly one of the two must
-be supplied.
+be supplied. If the user passes an empty string for either of the --pin or --triad
+arguments, the script will ignore that argument.
 
 Examples
 --------
 Generate two specific PINs:
-    $ ./scripts/generate_pinval.py \
+    $ python3 generate_pinval.py \
```

Comment:
Though both approaches should work, this interface is more similar to the way we recommend running Python scripts in other projects.